ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -247,14 +251,21 @@ const AccountListItem = ({
   };
 
   const getPreferredCurrencyValue = () => {
-    let value;
-    ///: BEGIN:ONLY_INCLUDE_IF(multichain)
-    value = account.balance;
-    ///: END:ONLY_INCLUDE_IF
-    ///: BEGIN:ONLY_INCLUDE_IF(build-main)
-    value = isEvmNetwork ? account.balance : balanceToTranslate;
-    ///: END:ONLY_INCLUDE_IF
-    return value;
+    const value = isEvmNetwork
+      ? stringifyBalance(
+          new BN(hexToDecimal(evmNativeTokenBalance[currentChainId])),
+          new BN(18),
```

Comment:
Are we sure about hardcoding this value of decimals? should come from the token itself?